exercises/practice/allergies/.docs/instructions.md

```diff
@@ -0,0 +1,27 @@
+# Instructions
+
+Given a person's allergy score, determine whether or not they're allergic to a given item, and their full list of allergies.
```

Comment:
So... We are not constructing a "list" but an unordered set.

I think in most languages (including Zig or just English) that "list" implies ordering. Ordering is going to imply data types that require allocating, but the problem itself doesn't necessitate ordering, we wouldn't care if a person is allergic to tomatoes _before_ cats or vice versa, just enough to answer yes/no questions.

I don't really know what to do with this. I think EnumSet is a good middle ground between readable code and a compact solution.

If I had requirements like this in the real world...
* If I thought I'd need to grow to support an arbitrary number of inputs, I'd be looking for a more dynamic solution that can scale for the future. Arrays or linked lists
* If I thought this was a fixed number of items, I might think about a much simpler solution of a bitmask and bitwise operators. I think this is what EnumSet is doing behind the scenes, but it's possible to remove a bit of code that way

Anyway, my recommendation is to keep EnumSet and change "list" to "set" in the instructions. In fact maybe a set makes more sense in most languages. For languages like Zig (or C, etc) I think a bitmask could also be acceptable

Reply:
Actually I take back the idea of a bitmask being the return value, since it's also the input value to `Allergen.list(u8)`

I think EnumSet is the best option